week0003/hoangdangduy/java/Customer.java

```diff
@@ -0,0 +1,35 @@
+public class Customer {
+	private String action;
```

Comment:
Customers must not know anything about actions! Actions must be put into another class!